test/Java8andUp/src/org/openj9/test/vmArguments/VmArgumentTests.java

```diff
@@ -145,9 +145,10 @@
 						"-Dsun.java.command",
 						"-Dsun.java.launcher"
 		};	
-		JAVA_VERSION=System.getProperty("java.version");
-		isJava8 = JAVA_VERSION.startsWith("1.8.0");
-		vmargsJarFilename = isJava8? "vmargs_SE80.jar": "vmargs_SE90.jar";
+		int javaMajorVersion = VersionCheck.major();
+		isJava8 = (8 == javaMajorVersion);
+		String javaMajorVersionString = "SE"+Integer.toString(javaMajorVersion)+"0";
+		vmargsJarFilename = "vmargs_"+javaMajorVersionString+".jar";
```

Comment:
I have recently understood that in a related PR it was mentioned that you do not need to add a version string to the jar file at all, as we only run one version or another, never at the same time.  Therefore, this could be simplified to simply :  vmargsJarFilename = "vmargs.jar";  

Please update the PR to do that.